geoapi/tests/api_tests/test_projects_service.py

```diff
@@ -116,11 +120,10 @@ def test_get_features_filter_type(projects_fixture,
 
 
 def test_update_project(projects_fixture):
-    user = db_session.query(User).get(1)
     data = {
         'name': 'new name',
         'description': 'new description'
     }
-    proj = ProjectsService.update(db_session, user, projects_fixture.id, data)
+    proj = ProjectsService.update(db_session, projects_fixture.id, data)
```

Comment:
Pedantic comment 9: For consistency, wouldn't it be better to use project instead of proj?

Reply:
We changed `proj` to `project` across the other files, same here for consistency?